retry.go

```diff
@@ -1,5 +1,26 @@
 package lo
 
+import (
+	"sync"
+	"time"
+)
+
+type debounce struct {
+	after time.Duration
+	mu    sync.Mutex
+	timer *time.Timer
+}
+
```

Comment:
Could you please add a `Cancel()` function ?

Reply:
Get it, I added `cancel` feature just now